app/src/components/main/modal/modalLayout/modalFooter/modalFooter.jsx

```diff
@@ -113,7 +111,14 @@ export class ModalFooter extends Component {
           )}
           {customButton && (
             <div className={cx('button-container')}>
-              <customButton.component {...customButton.buttonProps} onClick={onClickOk} />
+              {customButton.component ? (
+                <customButton.component
+                  {...customButton.buttonProps}
+                  onClick={() => customButton.onClick(onClickCancel)}
```

Comment:
Just pass `closeModal` to the `ModalFooter` as `closeHandler` prop and use it here instead of `onClickCancel`

Reply:
Done